docs/rules.md

```diff
@@ -269,6 +268,11 @@ This rule is to ensure the documentation link in the description does not contai
 
 Please refer to [doc-link-locale.md](./doc-link-locale.md) for details.
 
+### ResourceMustReferenceCommonTypes
+
+
```

Comment:
```suggestion

```

Reply:
This file is generated via `rush regen-ruleindex`. It looks like the script was picking up a local file I had. Regenerated to fix.